releasenotes/template.yaml

```diff
@@ -0,0 +1,36 @@
+# This YAML file describes the format for specifying a release notes entry for Istio.
+# This should be filled in for all user facing changes.
+
+# kind describes the type of change that this represents. 
+# Valid Values are:
+# - bug-fix -- Used to specify that this change represents a bug fix. 
+# - security-fix -- Used to specify that this change represents a security fix. 
+# - feature -- Used to specify a new feature that has been added. 
+# - test -- Used to describe additional testing added. This file is optional for
+# tests, but included for completeness.
+kind: 
+
+# area describes the area that this change affects. 
+# Valid values are:
+# - traffic-management
+# - security
+# - telemetry
+# - installation
+# - istioctl
+# - documentation
+area:
+
+# issue is a (potentially comma separated list) of GitHub issues resolved in this note.
+issue: 
+
+# releaseNotes is a markdown listing of any user facing changes. This will appear in the
+# release notes.
```

Comment:
Today we have a `Change Notes` and `Upgrade Notes` that is associated with each 1.x release (along with a page giving the overall release story) and then a `Changes` section on each patch release's page.

Do we need to emphasis where `releaseNotes` and `upgradeNotes` will be placed in this as opposed to just saying they will appear in the release notes? Or are we thinking the format may change some, so leaving it slightly ambiguous here is fine?

Reply:
I want to keep the format of the document independent of the YAML so think it's better to leave it ambiguous. We already have 1.x and 1.x.x with different formats and the format evolves in different releases.